src/Main.java

```diff
@@ -10,19 +11,12 @@ public static void main(String[] args) {
     Person d = new Person("Anne", LocalDate.parse("1969-03-15"), "+447220012222", "anne@gmail.com");
     Person e = new Person("Jack", LocalDate.parse("1996-08-20"), "+447220012098", "jack@gmail.com");
 
-
-    Person[] personArray = new Person[5];
-    personArray[0] = a;
-    personArray[1] = b;
-    personArray[2] = c;
-    personArray[3] = d;
-    personArray[4] = e;
-
+    List <Person> people = List.of(a,b,c,d,e);
```

Comment:
Nice change of the `List` name 👍 

Do you think you still need the variables for the `Person` objects added to the list? Or can you think of another way of adding them to the list without creating a variable?